subsys/bluetooth/controller/Kconfig

```diff
@@ -457,6 +457,11 @@ config BT_CTLR_ASSERT_HANDLER
 	  and will be invoked whenever the controller code encounters
 	  an unrecoverable error.
 
+config BT_CTRL_DF
```

Comment:
Typo: `CTRL -> CTLR`

Typically, a DF application only consist of parts of the entire DF specification. Therefore, the specification allows
to only qualify parts of it. See Core_v5.2, Vol 6, Section 4.6.
A suggested split of configurations is provided below.
* `BT_CTLR_DF_CONN_REQ`
* `BT_CTLR_DF_CONN_RSP`
* `BT_CTLR_DF_BROADCAST_RX`
* `BT_CTLR_DF_BROADCAST_TX`
* `BT_CTLR_DF_TX_AOA`
* `BT_CTLR_DF_RX`
* `BT_CTLR_DF_RX_AOD`

That will allow more fine grained control of which DF features that are included in the controller. 

To hide configurations that are not supported by a given controller, these configurations should have their corresponding `_SUPPORT` configurations which they depend upon. See for example `BT_CTLR_SLAVE_FEAT_REQ`.

Reply:
Configuration will be split into fine grained items that follow Core_v5.2, Vol 6, Section 4.6. DF configuration will be put in separate KConfig file and included in main controller KConfig.